### ISSUE TITLE
Multiple Files per Repo

### ISSUE THREAD

**Issue by ChristophLHR** [mc-cc-scripts/script-manager#1]
I like to split my User-Interfaces and my actual actions that run in the Background.
Therefore I’d like to have multiple files per Repo, which should be installed as in the folder structure.

**Comment by ColdIV** [comment]
My plan is to add some sort of info file to each repository.
Each repository must have this file.
The file will include relative paths to each file that it contains which should be downloaded.
For example, a program called `exampleProgram` would be within a repository called `exampleProgram-prog`. A possible file structure might look somewhat like this:

`exampleProgram-prog`
- `exampleProgram.lua`
- `infoFile`
- `exampleDependency1.lua`
- `includes/exampleDependency2.lua`

The `infoFile` should then look like this:
```
exampleProgram.lua
exampleDependency1.lua
includes/exampleDependency2.lua
``` 
So that the SCM can now download all those files by the paths given in the `infoFile`.
Everything else in the repository, like .gitignore or README.md, will not be touched.


All this is only a draft and may be implemented in a different way.

**Comment by ColdIV** [comment]
resolved in v1.0.0
